IRSA-README.md

```diff
@@ -0,0 +1,88 @@
+
+Such a hybrid architecture enables a wide variety of use-cases. For example, you could perform data-processing in the
+AWS cloud and store a training dataset in S3. Next you could trigger a ML Training job on a GPU on an on-premise 
+data plane for better cost performance. How should the on-premise data plane authenticate to AWS IAM to read data 
+from S3? Can the user assume an IAM role transparently based on policy defined in agreement between the Domino
+and the AWS Administration teams?
+
+It is also possible that two teams working on separate Domino data planes hosted on different Cloud Providers in the
+same organization may need to consume resources from each other. Data planes hosted in each of these Cloud Providers 
+may need to assume identities in the other Cloud Providers.  
+
+Using Web Identities based on an OpenID Connect Provider supports such use-cases. In this article we will start with a 
+basic setup where using IAM Role for Service Accounts (IRSA) we can configure Kubernetes Service Accounts attached to 
+an EKS workload to assume IAM Role(s). This is similar to attaching an IAM Role to an EC2 instance at startup. 
+Note that this mechanism allows EKS workloads to assume roles in any AWS Account and not just the AWS account hosting 
+the EKS cluster.
+
+In the subsequent series of articles we will expand on this to demonstrate how a data plane hosted on both on-premise 
+or any cloud provider can assume an IAM role in an AWS account.
+
+While this document is meant to guide you on configuring AWS IAM Role propagation to Domino Workloads based on 
+well defined policies consistent with your Enterprise Security guidelines, please speak with your Domino Professional 
+Services contact for additional information, assistance and support.
+
+### When should you consider using IRSA with Domino
+
+  - Does your Enterprise Policy forbid the use of long lived credentials?
+    
+  - Would you like to transparently and securely propagate IAM Role to all Domino Workloads including a 
+[Domino Scheduled Job](https://docs.dominodatalab.com/en/latest/user_guide/5dce1f/schedule-jobs/) ?
+    
+  - Would you like to configure a Domino Workload with a set of IAM Roles based on a set of complex custom criteria?
+
+
+### Why does using IRSA with Domino help achieve the above goals?
+
+  - When using the IRSA support in Domino, any Domino hosted workloads can assume IAM roles in any AWS account configured to
+    trust the Domino Platform
+    
+  - IRSA support in Domino allows configuration of multiple roles for a Domino workload where the boto3 library can 
+    switch across the multiple profiles configured in the ~/.aws/config file
+    
+  - The mapping of a workload to a set of roles can be configured based on custom configurable rules. This allows for 
+    complex mappings to be supported
+
+### How does IRSA work with Domino?
+ 1 . For a Domino Workload (implemented as a K8s Pod) to be able to assume an IAM Role, it must use the 
```

Comment:
Formatting of the bullets is a bit inconsistent (spaces at the start) but thats a minor thing

Reply:
Fixed